numpy/core/include/numpy/ndarrayobject.h

```diff
@@ -258,7 +258,7 @@ NPY_TITLE_KEY_check(PyObject *key, PyObject *value)
 }
 
 /* Macro, for backward compat with "if NPY_TITLE_KEY(key, value) { ..." */
-#define NPY_TITLE_KEY(key, value) (NPY_TITLE_KEY_check((key), (value)))
+#define NPY_TITLE_KEY(key, value) NPY_TITLE_KEY_check((key), (value))
```

Comment:
This breaks the backwards compat that is the reason for this existing in the first place

Reply:
As in users or downstream projects needs to be able to use the macro without parentheses in the C language?

Reply:
That's certainly what the comment above it implies, yes